.jenkins/Jenkinsfile

```diff
@@ -86,14 +85,11 @@ def checkPreCommitRequirements() {
                 timeout(15) {
                     // This is run to test that it works with the dependencies
                     // installed by our install-prereqs ansible playbook.
-                    sh './scripts/check-ci'
-                    // installed by our install-prereqs script.
 
                     dir('build') {
                         sh '''
                         cmake .. -DUSE_LIBSGX=OFF
                         make
-                        OE_SIMULATION=1 ctest --verbose --output-on-failure
```

Comment:
I am fine removing the tests part of this stage, as long as cmake and make are kept 👍